### DISCUSSION TITLE
Drag icon, is it possible to avoid the forbidden icon when draging over the new column position ?

### DISCUSSION THREAD

**Discussion by CamachoPt** [KevinVandy/material-react-table#358] [Q&A]
While drag and dropping a column, is it possible to change the forbidden icon when draging over the new column position ?

I tried the dropEffect select "move" and "copy" but they do not affect the cursor while dragging.

Ideally I wanted to "grabing" cursor to be mantained, I also tried to override the some css but was not successful.

**Comment by itshallrun** [comment]
@CamachoPt Have you found a solution to this?
